src/pages/AdminDashboard/AdminUsers.tsx

```
@@ -0,0 +1,86 @@
+        <Table data={data} tableHead={tableHead}>
+            {data && data.map((table:any)=>(
+                <tr key={table.id}>
+                    <td className='py-3 px-3 text-start'>{table.id}</td>
+                    <td>{table.name}</td>
+                    <td>{table.email}</td>
+                    <td>{table.status}</td>
+                    <td className=''>{table.action}</td>
+
+                </tr>
+
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)